docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
-TODO
+### Remove '-ComputerName' from *-Service cmdlets [5090](https://github.com/PowerShell/PowerShell/issues/5094)
+Although `-ComputerName` worked correctly, it was removed from `*-Service` cmdlets so that users consistently rely on PSRP and only have that port open.
 
 
 ### Unify cmdlets with parameter 'Encoding' to be of type System.Text.Encoding [5080](https://github.com/PowerShell/PowerShell/issues/5080)
-TODO
+The `-Encoding` Byte has been removed from the filesystem provider cmdlets. A new parameter -Byte is now the signal that a byte stream is required as input or output will be a stream of bytes.
 
 
 ### Add better error message for empty and null -UFormat arg [5055](https://github.com/PowerShell/PowerShell/issues/5055)
-TODO
+Previously, when passing an empty format string to `-UFormat`, a somewhat unhelpful error message would appear.  A more descriptive error has been added.
 
 
 ### Clean up console code [4995](https://github.com/PowerShell/PowerShell/issues/4995)
-TODO
-
-
-### correct passing of parameter so args show up as args instead of input [4963](https://github.com/PowerShell/PowerShell/issues/4963)
-TODO
-
-
-### remove -computer parameter from get-process [4960](https://github.com/PowerShell/PowerShell/issues/4960)
-TODO
+The following features were removed as they are not supported in PowerShell Core, and there are no plans to add support as they exist for legacy reasons for Windows PowerShell: psconsolefile switch and code, importsystemmodulues switch and code, and font changing code.
 
 
 ### removed RunspaceConfiguration support [4942](https://github.com/PowerShell/PowerShell/issues/4942)
-TODO
+In PSCore6 only InitialSessionState is supported.  Since the RunspaceConfiguration api was made internal, the code related to RunspaceConfiguration has been removed.  This means that some public apis have changed and it was decided that it was best to have a compile time error.
 
 
 ### Add Tests for *-computer cmdlets [4926](https://github.com/PowerShell/PowerShell/issues/4926)
 TODO
 
 
 ### CommandInvocationIntrinsics.InvokeScript bind arguments to $input instead of $args [4923](https://github.com/PowerShell/PowerShell/issues/4923)
-TODO
+An incorrect position of a parameter resulted in the args passed as input instead of as args.
 
 
 ### remove unsupported -showwindow switch from get-help [4903](https://github.com/PowerShell/PowerShell/issues/4903)
-TODO
+`-showwindow` relies on WPF, which is not supported on CoreCLR.
 
 
 ### allow * to be used in registry path for remove-item [4866](https://github.com/PowerShell/PowerShell/issues/4866)
-TODO
+Using `-literalpath` with an asterisk and the filesystem provider will now return an error rather than returning quietly.
 
 
 ### fix set-service failing test [4802](https://github.com/PowerShell/PowerShell/issues/4802)
-TODO
-
+`New-Service` should throw on System `-StartupType`
+TODO : Looks like this a change in test cases, not sure what the documentation process should be
```

Comment:
The change isn't only in test code.  Previously if you use `New-Service -StartupType foo`, I believe it quietly ignored `foo` and created the service with some default startup type.  The change is to explicitly throw an error for invalid startup type.